cmd/ir2proxy/main.go

```diff
@@ -26,46 +34,53 @@ func main() {
 	app.Parse(args)
 
 	if *yamlfile == "" {
-		app.FatalUsage("Need a YAML file to operate on.")
+		log.Error("Need a YAML file to operate on.")
+		app.Usage(args)
+		return 1
 	}
 
 	if !verifyYAMLFile(*yamlfile) {
-		log.Fatalf("File %s does not exist", *yamlfile)
+		log.Errorf("File %s does not exist", *yamlfile)
```

Comment:
Kingpin can do this for you

Reply:
I'm pretty sure there is a File type which specifies the argument provided matches a readable file